README.md

```diff
@@ -14,6 +14,150 @@ Takeaway Challenge
+
+2.6.5 :006 > maxs_mexican_shack.list_dishes
+Item: MexiCali Beef Burger -----> Price: 12
+Item: The Burrito Burger -----> Price: 15
+Item: El No Beef Burger -----> Price: 15
+Item: Los' Tres Bean Burger -----> Price: 11
+Item: Magic Bean Burger -----> Price: 11
+Item: Mexican Chik'n Burger -----> Price: 15
+Item: Stanky Fries -----> Price: 7
+Item: Nachos Libres -----> Price: 9
+
+2.6.5 :007 > maxs_mexican_shack.place_order
+Enter item off menu, when done type end order
+The Burrito Burger
+El No Beef Burger
+Stanky Fries
+end order
+ => ["The Burrito Burger ---> 15", "El No Beef Burger ---> 15", "Stanky Fries ---> 7"]
+2.6.5 :008 > maxs_mexican_shack.bill
+            The Burrito Burger ---> 15
+            El No Beef Burger ---> 15
+               Stanky Fries ---> 7
+                   ------------
+                        37
+
+2.6.5 :009 > maxs_mexican_shack.send_text
+ => <Twilio.Api.V2010.MessageInstance body: Sent from your Twilio trial account - Thank you! Your order has been placed
+      and will be delivered before 04:08 PM}>
+
+**Classes and Methods**
+
+There are 4 active classes
+
+* Dishes
+
+This contain the information for all the dishes the restaurant serves. It has 2 methods:
+
+* dishes
+This contain a hash of all the dishes and their prices.
+
+* list_dishes
+This return a list of all the dishes and their prices.
+
+* Order
+
+This contains the methods for ordering:
+
+* select_dishes
+This prompts the user to type in the dishes they want to order. It then matches them against dishes on the menu and pushes the matching ones into an array. It also pushes the monetary total into an array.
+
+* summarised_bill
+This returns a formatted bill which shows the menu items ordered with their prices and the total price for the order at the bottom.
+
+* Text
+
+This contains the information for sending a text out:
+
+* send_message
+Sends a message to a mobile number notifying the customer their order has been placed and how long it will take.
+
+* Restaurant
+
+This brings together the other classes and runs their methods using instances initialized on creating of a restaurant instance.
+
+* list_dishes
+Calls list dishes on an instance of Dishes class
+
+* place_order
+Calls select_dishes on an instance of the Order class
+
+* bill
+Calls summarised bill on an instance of Order class
+
+* send_text
+Calls send_message on an instance of Text class
+
+**RSpec**
+
+run 'RSpec' from the command line
+
+Finished in 0.05758 seconds (files took 1.26 seconds to load)
+
+12 examples, 0 failures
+
+
+COVERAGE:  97.62% -- 123/126 lines in 8 files
+
++----------+-------------+-------+--------+-----------+
+| coverage | file        | lines | missed | missing   |
++----------+-------------+-------+--------+-----------+
+|  75.00%  | lib/text.rb | 12    | 3      | 12, 22-23 |
++----------+-------------+-------+--------+-----------+
+7 file(s) with 100% coverage not shown
+
+**Learns**
+
+If i had more time I would do the following:
```

Comment:
Great section, rate this. Actually overall your readme is brilliant. Good clarity, can see your program broken down, user experience and intended direction.